cmd/kubernetes/operator/kodata/hub/v1.4.0/db-migration/db-migration.yaml

```diff
@@ -0,0 +1,52 @@
+# Copyright © 2020 The Tekton Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: tekton-hub-db-migration
+  namespace: tekton-pipelines
+  labels:
+    app: db
+spec:
+  template:
+    spec:
+      containers:
+        - name: db-migration
+          image: quay.io/tekton-hub/db-migration:v1.4.0
+          env:
+            - name: POSTGRES_HOST
+              value: tekton-hub-db
```

Comment:
im assuming that we will remove these yaml files from this patch and extend the [fetch releases script](https://github.com/tektoncd/operator/blob/main/hack/fetch-releases.sh)

Reply:
yes